src/AppRoutes.js

```diff
@@ -7,14 +7,14 @@ import PropTypes from 'prop-types';
 import { TabLayout } from './layouts';
 import { SignIn as SignInView, AccessDenied as AccessDeniedView } from './views';
 import Workspaces from './views/Workspaces';
-import { useWorkspacesList } from './state/hooks/WorkspaceHooks';
+import { useWorkspace } from './state/hooks/WorkspaceHooks';
 
 const AppRoutes = (props) => {
   const { authenticated, authorized, tabs } = props;
   const location = useLocation();
   const providedUrl = sessionStorage.getItem('providedUrl');
   const providedUrlBeforeSignIn = sessionStorage.getItem('providedUrlBeforeSignIn');
-  const workspacesList = useWorkspacesList();
+  const currentWorkspace = useWorkspace();
```

Comment:
If we don't use `workspace.status`, we can use the `useWorkspaceData` hook instead :slightly_smiling_face:

Reply:
good idea, thanks !